host/sgx/create.c

```diff
@@ -1021,7 +1042,7 @@ oe_result_t oe_create_enclave(
 {
     oe_result_t result = OE_UNEXPECTED;
     oe_enclave_t* enclave = NULL;
-    oe_sgx_load_context_t context;
+    oe_sgx_load_context_t context = {0};
```

Comment:
Are you getting a compiler warning for this? `oe_sgx_load_context_t` has nested structures and arrays and IIRC, C would require that the initializer for the appropriate max depth e.g. `{{0}}` or however deeply the structure is nested.

Reply:
Just checked, `{ 0 }` also initializes the nested struct. However, it seems unnecessary to initialize `context`.